cmd/ls.go

```diff
@@ -0,0 +1,237 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"reanahub/reana-client-go/client"
+	"reanahub/reana-client-go/client/operations"
+	"reanahub/reana-client-go/utils"
+
+	"github.com/go-gota/gota/dataframe"
+	"github.com/go-gota/gota/series"
+	"github.com/spf13/viper"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+const lsDesc = `
+List workspace files.
+
+The ` + "``ls``" + ` command lists workspace files of a workflow specified by the
+environment variable REANA_WORKON or provided as a command-line flag
+` + "``--workflow`` or ``-w``." + ` The SOURCE argument is optional and specifies a
+pattern matching files and directories.
+
+Examples:
+
+  $ reana-client ls --workflow myanalysis.42
+
+  $ reana-client ls --workflow myanalysis.42 --human-readable
+
+  $ reana-client ls --workflow myanalysis.42 'data/*root*'
+
+  $ reana-client ls --workflow myanalysis.42 --filter name=hello
+`
+
+const lsFormatFlagDesc = `
+Format output according to column titles or column
+values. Use <column_name>=<column_value> format.
+E.g.display files named data.txt
+--format name=data.txt
+`
```

Comment:
I think we should remove the newline at the beginning and at the end of this string, so that they are not present in the help message. What do you think?